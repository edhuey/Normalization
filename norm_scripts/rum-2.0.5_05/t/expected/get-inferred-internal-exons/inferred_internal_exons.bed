track	name="Inferred Internal Exons"  description="Inferred Internal Exons"   visibility=3    itemRgb="On"
1	14831081	14831467	.	0	.	14831081	14831467	0,205,102
1	15083982	15084444	.	0	.	15083982	15084444	0,205,102
1	16596639	16596658	.	0	.	16596639	16596658	0,205,102
2	3615287	3615633	.	0	.	3615287	3615633	0,205,102
2	3615989	3616215	.	0	.	3615989	3616215	0,205,102
2	3617283	3617546	.	0	.	3617283	3617546	0,205,102
2	3617283	3617461	.	0	.	3617283	3617461	0,205,102
2	3617724	3618199	.	0	.	3617724	3618199	0,205,102
2	3618229	3618635	.	0	.	3618229	3618635	0,205,102
2	3618406	3618635	.	0	.	3618406	3618635	0,205,102
2	3618406	3618852	.	0	.	3618406	3618852	0,205,102
2	3618554	3618852	.	0	.	3618554	3618852	0,205,102
2	3618554	3618635	.	0	.	3618554	3618635	0,205,102
2	3619209	3619579	.	0	.	3619209	3619579	0,205,102
2	3619935	3620384	.	0	.	3619935	3620384	0,205,102
2	3619935	3620090	.	0	.	3619935	3620090	0,205,102
2	3619974	3620384	.	0	.	3619974	3620384	0,205,102
2	3619974	3620090	.	0	.	3619974	3620090	0,205,102
2	3621518	3621795	.	0	.	3621518	3621795	0,205,102
2	3621518	3621735	.	0	.	3621518	3621735	0,205,102
2	3622091	3622546	.	0	.	3622091	3622546	0,205,102
3	13590496	13590723	.	0	.	13590496	13590723	0,205,102
3	13590496	13590684	.	0	.	13590496	13590684	0,205,102
3	13590545	13590684	.	0	.	13590545	13590684	0,205,102
3	13590545	13590723	.	0	.	13590545	13590723	0,205,102
3	13591040	13591178	.	0	.	13591040	13591178	0,205,102
3	13591079	13591178	.	0	.	13591079	13591178	0,205,102
3	14192315	14192637	.	0	.	14192315	14192637	0,205,102
3	14192315	14192664	.	0	.	14192315	14192664	0,205,102
3	14194659	14195083	.	0	.	14194659	14195083	0,205,102
3	14194839	14195083	.	0	.	14194839	14195083	0,205,102
4	3951874	3952181	.	0	.	3951874	3952181	0,205,102
4	3951874	3952082	.	0	.	3951874	3952082	0,205,102
4	3953070	3953426	.	0	.	3953070	3953426	0,205,102
4	3953070	3953413	.	0	.	3953070	3953413	0,205,102
4	3953426	3953739	.	0	.	3953426	3953739	0,205,102
4	3953591	3953739	.	0	.	3953591	3953739	0,205,102
4	3953604	3954104	.	0	.	3953604	3954104	0,205,102
4	3953604	3953739	.	0	.	3953604	3953739	0,205,102
4	3954316	3954446	.	0	.	3954316	3954446	0,205,102
4	3954624	3954968	.	0	.	3954624	3954968	0,205,102
4	3954629	3954968	.	0	.	3954629	3954968	0,205,102
4	3955619	3955752	.	0	.	3955619	3955752	0,205,102
5	11701924	11702127	.	0	.	11701924	11702127	0,205,102
5	11705281	11705433	.	0	.	11705281	11705433	0,205,102
5	11708672	11708925	.	0	.	11708672	11708925	0,205,102
5	11708672	11708816	.	0	.	11708672	11708816	0,205,102
5	11708672	11708837	.	0	.	11708672	11708837	0,205,102
5	11710350	11710406	.	0	.	11710350	11710406	0,205,102
5	11710380	11710406	.	0	.	11710380	11710406	0,205,102
5	11723643	11723756	.	0	.	11723643	11723756	0,205,102
5	11723836	11723853	.	0	.	11723836	11723853	0,205,102
5	11724208	11724708	.	0	.	11724208	11724708	0,205,102
5	11724466	11724708	.	0	.	11724466	11724708	0,205,102
5	11726673	11727155	.	0	.	11726673	11727155	0,205,102
5	11726673	11726872	.	0	.	11726673	11726872	0,205,102
5	11727228	11727533	.	0	.	11727228	11727533	0,205,102
5	11727228	11727360	.	0	.	11727228	11727360	0,205,102
5	11727538	11727550	.	0	.	11727538	11727550	0,205,102
5	11728044	11728439	.	0	.	11728044	11728439	0,205,102
5	11728244	11728439	.	0	.	11728244	11728439	0,205,102
5	11728599	11728745	.	0	.	11728599	11728745	0,205,102
5	11728923	11729159	.	0	.	11728923	11729159	0,205,102
5	11731739	11732095	.	0	.	11731739	11732095	0,205,102
5	11731818	11732095	.	0	.	11731818	11732095	0,205,102
5	11732273	11732491	.	0	.	11732273	11732491	0,205,102
5	11732273	11732334	.	0	.	11732273	11732334	0,205,102
5	11732273	11732352	.	0	.	11732273	11732352	0,205,102
5	11732512	11732779	.	0	.	11732512	11732779	0,205,102
5	11732512	11732590	.	0	.	11732512	11732590	0,205,102
5	11732512	11732664	.	0	.	11732512	11732664	0,205,102
5	11732530	11732779	.	0	.	11732530	11732779	0,205,102
5	11732530	11732590	.	0	.	11732530	11732590	0,205,102
5	11732530	11732664	.	0	.	11732530	11732664	0,205,102
5	11733098	11733595	.	0	.	11733098	11733595	0,205,102
5	11734750	11735231	.	0	.	11734750	11735231	0,205,102
5	11735169	11735615	.	0	.	11735169	11735615	0,205,102
5	11735169	11735231	.	0	.	11735169	11735231	0,205,102
5	11735169	11735483	.	0	.	11735169	11735483	0,205,102
5	11735194	11735615	.	0	.	11735194	11735615	0,205,102
5	11735194	11735483	.	0	.	11735194	11735483	0,205,102
5	11735194	11735231	.	0	.	11735194	11735231	0,205,102
