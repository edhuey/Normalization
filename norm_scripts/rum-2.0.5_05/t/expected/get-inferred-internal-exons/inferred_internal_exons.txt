1:14831082-14831467	novel
1:15083983-15084444	novel
1:16596640-16596658	novel
2:3615288-3615633	novel
2:3615990-3616215	novel
2:3617284-3617546	novel
2:3617284-3617461	novel
2:3617725-3618199	novel
2:3618230-3618635	novel
2:3618407-3618635	novel
2:3618407-3618852	novel
2:3618555-3618852	novel
2:3618555-3618635	novel
2:3619210-3619579	novel
2:3619936-3620384	novel
2:3619936-3620090	novel
2:3619975-3620384	novel
2:3619975-3620090	novel
2:3621519-3621795	novel
2:3621519-3621735	novel
2:3622092-3622546	novel
3:13590497-13590723	novel
3:13590497-13590684	novel
3:13590546-13590684	novel
3:13590546-13590723	novel
3:13591041-13591178	novel
3:13591080-13591178	novel
3:14192316-14192637	novel
3:14192316-14192664	novel
3:14194660-14195083	novel
3:14194840-14195083	novel
4:3951875-3952181	novel
4:3951875-3952082	novel
4:3953071-3953426	novel
4:3953071-3953413	novel
4:3953427-3953739	novel
4:3953592-3953739	novel
4:3953605-3954104	novel
4:3953605-3953739	novel
4:3954317-3954446	novel
4:3954625-3954968	novel
4:3954630-3954968	novel
4:3955620-3955752	novel
5:11701925-11702127	novel
5:11705282-11705433	novel
5:11708673-11708925	novel
5:11708673-11708816	novel
5:11708673-11708837	novel
5:11710351-11710406	novel
5:11710381-11710406	novel
5:11723644-11723756	novel
5:11723837-11723853	novel
5:11724209-11724708	novel
5:11724467-11724708	novel
5:11726674-11727155	novel
5:11726674-11726872	novel
5:11727229-11727533	novel
5:11727229-11727360	novel
5:11727539-11727550	novel
5:11728045-11728439	novel
5:11728245-11728439	novel
5:11728600-11728745	novel
5:11728924-11729159	novel
5:11731740-11732095	novel
5:11731819-11732095	novel
5:11732274-11732491	novel
5:11732274-11732334	novel
5:11732274-11732352	novel
5:11732513-11732779	novel
5:11732513-11732590	novel
5:11732513-11732664	novel
5:11732531-11732779	novel
5:11732531-11732590	novel
5:11732531-11732664	novel
5:11733099-11733595	novel
5:11734751-11735231	novel
5:11735170-11735615	novel
5:11735170-11735231	novel
5:11735170-11735483	novel
5:11735195-11735615	novel
5:11735195-11735483	novel
5:11735195-11735231	novel
